hudi-common/src/main/java/org/apache/hudi/common/model/HoodiePayloadProps.java

```diff
@@ -40,4 +40,12 @@
    */
   public static final String PAYLOAD_EVENT_TIME_FIELD_PROP = "hoodie.payload.event.time.field";
   public static String DEFAULT_PAYLOAD_EVENT_TIME_FIELD_VAL = "ts";
+
+  public static final String PAYLOAD_DELETE_CONDITION = "hoodie.payload.delete.condition";
```

Comment:
would you please add some docs about the added fields?

Reply:
ok